.github/workflows/ci.yml

```diff
@@ -27,8 +27,25 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           targets: ${{ matrix.target }}
-      - run: cargo build --target ${{ matrix.target }} --no-default-features
-      - run: cargo build --target ${{ matrix.target }}
+      - uses: taiki-e/install-action@cargo-hack
+      - run: cargo hack build --target ${{ matrix.target }} --feature-powerset
+
+  build-feature-powerset:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.83.0
```

Comment:
We'll want 1.85 so we get the 2024 edition perhaps?

Reply:
`crypto-bigint=0.6` has MSRV 1.83, we'll bump when we switch to 0.7.